src/compiler.jl

```diff
@@ -4044,6 +4044,15 @@ function GPUCompiler.codegen(output::Symbol, job::CompilerJob{<:EnzymeTarget};
                                   ])
             continue
         end
+        # Since this is noreturn, it can't write to any operations in the function
+        # in a way accessible by the function. Ideally the attributor should actually
```

Comment:
This is not generally true for no-return functions in the presence of longjump/setjmp. It is true for this function